deltachat-ios/Controller/WebxdcViewController.swift

```diff
@@ -62,24 +62,37 @@ class WebxdcViewController: WebViewViewController {
         
         let script = """
         window.webxdc = (() => {
-          let setUpdateListenerPromise = null
           var log = (s)=>webkit.messageHandlers.log.postMessage(s);
         
           var update_listener = () => {};
 
-          window.__webxdcUpdate = async (lastSerial) => {
+          let should_run_again = false;
+          let running = false;
+          let lastSerial = 0;
+          window.__webxdcUpdate = async () => {
+            if (running) {
+                should_run_again = true
+                return
+            }
+            if (should_run_again) {
+                should_run_again = false
+            }
```

Comment:
```suggestion
            should_run_again = false
```

i think, the condition is not needed.

Reply:
yes, you are correct here, thanks